build.gradle

```diff
@@ -100,7 +100,6 @@ allprojects {
         force "io.swagger:swagger-annotations:$swaggerVersion"
         force "com.google.protobuf:protobuf-java:$protobufVersion"
         force "com.google.guava:guava:$guavaVersion"
-        force "junit:junit:$junitVersion"
```

Comment:
This is needed, unfortunately, because of the CVE fix. Please revert.

Reply:
Thanks. Reverted.